ujt/utils.py

```diff
@@ -68,3 +69,15 @@ def read_proto_from_file(file_name: str, proto_type: Type[Message]) -> Message:
               "r") as f:
         proto_text: str = f.read()
         return text_format.Parse(proto_text, proto_type())
+
+
+def is_client_cytoscape_node(tap_node):
```

Comment:
I have a few more utility functions now -- we could split this file into proto_utils and utils if necessary? @KenArthur